### ISSUE TITLE
disableKinds change has caused the example site to not work well

### ISSUE THREAD

**Issue by theodore-dream** [serg/yourfolio#6]
A few notes from trying this on Linux with following extended binary 

Hugo Static Site Generator v0.74.2-48565DE6/extended linux/amd64 BuildDate: 2020-07-17T17:32:27Z

 - it definitely doesn't work out of the box, but as implied in the main repo you need to use the exampleSite to test out this theme, ok

Here's the error I see trying the quickstart guide and seeing if it works, nope

```
hugo server
Building sites … WARN 2020/07/18 20:47:46 found no layout file for "HTML" for kind "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2020/07/18 20:47:46 found no layout file for "HTML" for kind "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
Built in 12 ms
Error: Error building site: failed to render pages: render of "home" failed: execute of template failed: template: index.html:3:6: executing "main" at <partial "navigation" .>: error calling partial: "/home/admin/Documents/webdev/git-sites/hugotest/quickstart/themes/yourfolio/layouts/partials/navigation.html:2:25": execute of template failed: template: partials/navigation.html:2:25: executing "partials/navigation.html" at <len .Site.Menus.main>: error calling len: reflect: call of reflect.Value.Type on zero Value
```

Then I google the error and find a thread showing this refers to a type of content that is failing, so I check out the config.yaml site from exampleSite, ah I see some changes, looks like disableKinds line is what I need, so I try it out, however after adding this line

```
disableKinds: [RSS, taxonomy, taxonomyTerm]
```

However I see an error that explains that Hugo 0.73.0 made a change to the behavior of "taxonomy" value, but I don't have enough debugging skill to understand what different paramter I'm supposed to use instead of "taxonomy" 

```
hugo server
ERROR 2020/07/18 21:00:56 You have the value 'taxonomy' in the disabledKinds list. In Hugo 0.73.0 we fixed these to be what most people expect (taxonomy and term).
But this also means that your site configuration may not do what you expect. If it is correct, you can suppress this message by following the instructions below.
If you feel that this should not be logged as an ERROR, you can ignore it by adding this to your site config:
ignoreErrors = ["error-disable-taxonomy"]
Building sites … WARN 2020/07/18 21:00:56 found no layout file for "HTML" for kind "section": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
Built in 12 ms
Error: Error building site: failed to render pages: render of "home" failed: execute of template failed: template: index.html:3:6: executing "main" at <partial "navigation" .>: error calling partial: "/home/admin/Documents/webdev/git-sites/hugotest/quickstart/themes/yourfolio/layouts/partials/navigation.html:2:25": execute of template failed: template: partials/navigation.html:2:25: executing "partials/navigation.html" at <len .Site.Menus.main>: error calling len: reflect: call of reflect.Value.Type on zero Value
```

At this point I've pretty much given up on trying to use this theme as it seems outdated and partially broken, but it is beautiful and I love the example site, so I may come back to this, hoping serg might try and help fix this to keep this theme up to date, if interested. Or perhaps I haven't followed the exampleSite properly, in which case I do apologize if I've missed something, but I do feel I've spent a decent bit of time trying to set this up and either:

 - the setup instructions are sub-par in my opinion, OR
 - due to changes in Hugo and how disableKinds parameter (or some other config paramter) works, this theme is not up to date and cannot be used "out of the box" 

Cheers

**Comment by serg** [comment]
Hello @theodore-dream 
Thank you for your interest to my theme!
Let me help you to run this theme.

As you may heard Hugo is evolve fast and new versions came out is too often. But, Hugo is improving from version to version, sometime with some breakable things, those not supported in new versions. This is normal development process.

The last version, where I tested theme was 0.72/extended. Some issues appeared and related with renaming for taxonomy terms in 0.73 version. Here is some info about [this](https://github.com/gohugoio/hugo/issues/6911).
For fixing, you need replace this:
```
disableKinds: [RSS, taxonomy, taxonomyTerm]
```
with:
```
disableKinds: [RSS, term, taxonomy]
```
The minimum requirement for this theme is 0.64.1 for Hugo version. I will need to think a little bit about the update above for make opportunity to build this theme on the few versions of Hugo before 0.73. I think the best way is mention this in readme.

Also, thank you for pointing me up about adding updates to readme file. I will add the updates about how to build the theme and keynotes about how to use theme (here is some of helpful things, those I would like to describe) when have a time.

Here is a quick optional tip about how to check theme by using only cloned version of this repository (see more info [here](https://discourse.gohugo.io/t/how-to-check-theme-locally-without-creating-a-hugo-project/12422)). Inside cloned folder you could run this command:
```
hugo server -s exampleSite --themesDir=../..
```

Please let me know if you got a successful build on your side after applying the updates above or info about any other concerns with build.

**Comment by serg** [comment]
@theodore-dream, also, according to your logs, seems like you have issues in menu part for your version of theme. Do you have any menu items, defined in your `config.yaml` file?

**Comment by theodore-dream** [comment]
Hi @serg I completely understand apologies if I came across harsh in any way just wanted to bring this to your attention. I am completely new to Hugo but pretty familiar with Linux and thought the error was pretty clear so thought I'd share. But I wasn't aware how quickly Hugo is changing so I can understand its a lot of work to keep a theme up to date.

I did test your fix on my example site, and it did run without the same error but the site was completely blank as obviously I was missing necessary content for the theme. My example site was just testing the functionality I don't have any actual site content built right now just testing. So no my content would not have showed anything even if it did work successfully without error, I'm now realizing. Makes sense.

I did try it again using your exampleSite and with your updated disablekinds parameter it did work! I do like your tip on how to quickly test. 

This is resolved, just a few thoughts/realizations
 - Without the user of the theme or the main owner having a very high level of involvement in the Hugo project and all its updates, it is very hard to sustain a theme's development and keep in line with the upstream, I guess that's just life. 
 - Taking a closer look at your _index.md it does seem you have to manually set the image sizes, so not sure how much of a learning curve is there, but this is definitely my favourite functionality of your theme and I'm honestly a bit worried this would break over time if I were to keep updating Hugo version, I don't know enough to say how "standard" many of your unique pieces are (the highlighting of the images) 

Basically it *seems* (not enough expertise to really know for sure) that your theme's uniqueness is a double edged sword, because it simultaneously makes it very attractive because I like the unique elements, but those unique elements and lack of folks using them in this other/themes (from what I can tell) also means it seems likely that Hugo updates may break that functionality over time. So to be completely honest this makes me want to continue my search for other themes. Bittersweet. Not 100% sure just sharing my thoughts. Really appreciate the prompt feedback regardless Serg!

**Comment by serg** [comment]
@theodore-dream, thank you for your feedback! Unfortunately I can't reproduce this issue with default content (provided in theme). I sent you a message to your email with instructions (2 min video) how to install theme on your side. Hope it will be helpful.

**Comment by theodore-dream** [comment]
@serg yes this issue is resolved, allow me to show you the reproduction steps. This is basically to explain why you must use the exampleSite you provide and my failure to do this caused the issue, I was trying the example steps from Hugo quick start https://gohugo.io/getting-started/quick-start/ and this is what lead to the issue.

So it seems that this is not an issue of one simple config field as I misunderstood, but an overall misunderstanding on my part regarding how this is meant to be setup, and the user of this theme has to use the exampleSite and build from there. 

----
As example please see on the exampleSite folder inside the theme folder for how to configure a website and create content files.
---
^^^ I should have paid more attention to this

Reproduction steps for issue: 
 1179  hugo new site quickstart
 1180  cd quickstart/
 1181  git init
 1182  git submodule add https://github.com/serg/yourfolio.git themes/yourfolio
 1183  echo 'theme = "yourfolio"' >> config.toml 
 1184  hugo new posts/my-first-post.md
 1185  hugo server -D

hugo server -D
Building sites … WARN 2020/07/19 19:30:39 found no layout file for "HTML" for kind "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2020/07/19 19:30:39 found no layout file for "HTML" for kind "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2020/07/19 19:30:39 found no layout file for "HTML" for kind "section": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
ERROR 2020/07/19 19:30:39 render of "home" failed: execute of template failed: template: index.html:3:6: executing "main" at <partial "navigation" .>: error calling partial: "/home/admin/Documents/webdev/git-sites/hugotest/quickstart/themes/yourfolio/layouts/partials/navigation.html:2:25": execute of template failed: template: partials/navigation.html:2:25: executing "partials/navigation.html" at <len .Site.Menus.main>: error calling len: reflect: call of reflect.Value.Type on zero Value
Built in 13 ms
Error: Error building site: failed to render pages: render of "page" failed: execute of template failed: template: _default/single.html:3:6: executing "main" at <partial "navigation" .>: error calling partial: "/home/admin/Documents/webdev/git-sites/hugotest/quickstart/themes/yourfolio/layouts/partials/navigation.html:2:25": execute of template failed: template: partials/navigation.html:2:25: executing "partials/navigation.html" at <len .Site.Menus.main>: error calling len: reflect: call of reflect.Value.Type on zero Value

However I can confirm that yes this works perfectly using your exampleSite as demonstrated in your video!
